packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
-import { BaseComponent, KeyCodes, css, getId, getRTL, getRTLSafeKeyCode } from '../../Utilities';
-import { ISliderProps, ISlider, ISliderStyleProps, ISliderStyles } from './Slider.types';
-import { classNamesFunction, getNativeProps, divProperties } from '../../Utilities';
-import { Label } from '../../Label';
-
+import { BaseComponent, KeyCodes, css, getId, getRTL, getRTLSafeKeyCode } from 'office-ui-fabric-react/lib/Utilities';
+import { ISliderProps, ISlider, ISliderStyleProps, ISliderStyles, ISliderMarksArrayFormat } from './Slider.types';
+import { classNamesFunction, getNativeProps, divProperties } from 'office-ui-fabric-react/lib/Utilities';
+import { Label } from 'office-ui-fabric-react/lib/Label';
+import { TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
```

Comment:
How did this all imports got changed? Revert please.

Reply:
I think these were throwing errors about being relative imports with the old path format.

Reply:
ok reverted